apps/prairielearn/src/pages/instructorQuestions/instructorQuestions.html.ts

```diff
@@ -0,0 +1,45 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { QuestionsTable } from '../../components/QuestionsTable.html';
+import { compiledScriptTag } from '../../lib/assets';
+import { QuestionsTableData } from '../../models/questions';
+
+export const QuestionsPage = ({
+  questions,
+  showAddQuestionButton,
+  resLocals,
+}: {
+  questions: QuestionsTableData[];
+  showAddQuestionButton: boolean;
+  resLocals;
+}) => {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../../pages/partials/head') %>", resLocals)}
+      </head>
+
+      <body>
+        ${renderEjs(__filename, "<%- include('../partials/navbar'); %>", resLocals)}
+        ${compiledScriptTag('instructorQuestionsClient.ts')}
```

Comment:
This should probably be in the head, not the body.

Reply:
I've moved the script tag to `head`. 

For `partials/navbar`, across at least a few other pages I looked at, this is but in the body, but outside of main. I am going to leave it that way for consistency. If we think this should be changed, its a conversation we can have but it should be done across all pages for consistency.